src/adapters/sqlite/encodeSchema/index.js

```diff
@@ -55,6 +151,12 @@ const encodeAddColumnsMigrationStep: AddColumnsMigrationStep => SQL = ({ table,
       )} = ${encodeValue(nullValue(column))};`
       const addIndex = encodeIndex(column, table)
 
+      if (column.isSearchable) {
+        logger.warn(
+          '[DB][Worker] Support for migrations and isSearchable is still to be implemented',
+        )
```

Comment:
If there's no support for schema migrations for FTS, that's fine, I can merge that - but throw an error if someone tries to do that - not just on add columns, but for create table migration as well